### PR TITLE
liferay lexicon

### PR DESCRIPTION
**Pull request by victorvalle**

### CODE REVIEW

File: src/data/systems/liferay.JSON [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    "url": "",
+    "label": "code documentation"
+  },
+  "storybook": {
+    "data": "no data",
+    "url": "",
+    "label": "storybook"
+  },
+  "distribution": {
+    "data": "npm, Lerna",
```
Comment (michalstocki) [inline src/data/systems/liferay.JSON]: This should be:
```
"data": [
      "npm",
      "Lerna"
],
```

**Comment by marcintreder** [comment]
@victorvalle thank you for your contribution! I've added an array of data for distribution (whenever there's more than one option keep strings in an array) and I removed 'implementation' part of the JSON. It would be tough to find similar data for others systems in Adele which would result in 'no data' for dozens of systems.

**Comment by victorvalle** [comment]
Others also have it. They separate system from the implementation. Like BBC as an example.

**Comment by marcintreder** [comment]
Interesting! Could you show me some examples? If we can find couple of examples for other systems, we can surely update the data :). 

ps. Lexicon is so cool 😍

**Pull request merged**